config/locales/es/activerecord.yml

```diff
@@ -302,6 +302,7 @@ es:
         body: "Mensaje"
         title: "Título"
       signature_sheet:
+        title: "Titulo"
```

Comment:
There's a small typo here; correct Spanish is "Título" (you can copy the text two lines above this one if your keyboard doesn't generate Spanish characters :wink:).